frameworks/elastic/docs/upgrade.md

```diff
@@ -21,3 +21,111 @@ $ dcos package install elastic --options=/path/to/options.json
 The command above will trigger the install of the new Elastic version. You can follow the upgrade progress by making a REST request identical to the one used to follow the progress of a configuration upgrade. See the Configuring section for more information.
 
 Note: The upgrade process will cause all of your Elastic node processes to restart.
+
+## Rolling Upgrades
+
+By default, the update strategy is `serial`, which waits for each node to be ready before moving on to the next one. But the elasticsearch node being ready doesn't mean all the shards in the indexes are ready. This may be an appropriate strategy for test clusters or clusters where writes can be suspended during the upgrade/update process, but for production clusters you may want to use the `safe` strategy.
```

Comment:
Should `elasticsearch` be capitalized?

Suggest:

The default upgrade strategy is `serial`. This strategy waits for each node to be ready before moving on to the next one. 

However, a node can be ready before all of the shards in the indexes are ready. For this reason, the `serial` upgrade strategy is only appropriate for test clusters or clusters where writes can be suspended during the upgrade/update process. For production clusters, use the `safe` strategy.